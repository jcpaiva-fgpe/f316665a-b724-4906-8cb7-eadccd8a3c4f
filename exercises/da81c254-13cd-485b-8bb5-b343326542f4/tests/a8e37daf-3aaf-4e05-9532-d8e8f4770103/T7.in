19 59
20 0
