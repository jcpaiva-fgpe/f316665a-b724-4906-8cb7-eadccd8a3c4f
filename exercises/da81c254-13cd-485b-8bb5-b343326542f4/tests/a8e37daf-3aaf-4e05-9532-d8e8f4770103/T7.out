Passou apenas 1 minuto!
De facto!
